lib/admin/WP_Auth0_Admin_Advanced.php

```diff
@@ -556,8 +556,6 @@ public function connections_validation( $old_options, $input ) {
 
     if ($input['passwordless_enabled'] && $input['passwordless_enabled'] != $old_options['passwordless_enabled']) {
 
-      // $check_if_enabled = explode(',', $input['lock_connections']);
```

Comment:
what happened to all these comments that are now removed? was it implemented somewhere?

Reply:
They've been commented for a while

Reply:
but is it something you want to do someday? are you tracking it somewhere else or are they just gone for good? 😂

Reply:
I'm not sure what it's for, it's obviously not doing anything, and it's related to the settings I'm working with so 👋 

:)